<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>1476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>627</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	padding-left: 10px;
	background-color: rgb(225, 225, 225);
}</string>
     </property>
     <property name="text">
      <string>Номер заказа</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea {
	padding-left: 20px;
	padding-right: 7px;
	background-color: rgb(255, 255, 255);
}

QScrollBar:vertical {
	border: none;
	margin-top: 5px;
	margin-bottom: 5px;
	background: rgb(255, 255, 255);
	width: 7px;
}

QScrollBar::handle:vertical {
	background-color: rgb(180, 180, 180);
	min-height: 30px;
	border-radius: 3px;
}

QScrollBar::handle:vertical:hover {
	background-color: rgb(255, 145, 67);
}

QScrollBar::handle:vertical:pressed {
	background-color: rgb(223, 124, 59);
}

QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(255, 255, 255);
	height: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(255, 255, 255);
	height: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>593</width>
        <height>1462</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	margin-right: 7px;
}

QScrollArea, QFrame, QLabel, QWidget#scrollAreaWidgetContents, QWidget#scrollAreaWidgetContents_2, QWidget#scrollAreaWidgetContents_3 {
	background-color: rgb(255, 255, 255);
}

QLabel {
	margin-top: 10px;
}

	</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(234, 234, 234);
}</string>
         </property>
         <property name="text">
          <string>Выбранные детали</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Название</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Длина</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ширина</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Толщина</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Количество</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Приоритет</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Статус</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(234, 234, 234);
}</string>
         </property>
         <property name="text">
          <string>Выбранные слитки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea {
	padding: 0px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>577</width>
            <height>144</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(234, 234, 234);
}</string>
         </property>
         <property name="text">
          <string>Остатки со слитков</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea {
	padding: 0px;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>577</width>
            <height>144</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="detailedPlanFrame">
         <property name="styleSheet">
          <string notr="true">* {
	margin-right: 0px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	background-color: rgb(234, 234, 234);
}</string>
            </property>
            <property name="text">
             <string>Карта раскроя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fullScreen">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	margin-top: 10px;
	border: none;
	background-color: rgb(234, 234, 234);
}

QPushButton:hover {
	background-color: rgb(216, 216, 216);
}

QPushButton:pressed {
	background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>На весь экран</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="detailedPlan">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	margin-top: 10px;
	margin-right: 7px;
	border: none;
	background-color: rgb(234, 234, 234);
}

QPushButton:hover {
	background-color: rgb(216, 216, 216);
}

QPushButton:pressed {
	background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>Перейти к плану</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::ScrollHandDrag</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: rgb(234, 234, 234);
}</string>
         </property>
         <property name="text">
          <string>Статистика по заказу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Выход годного:

Готовых деталей:
Неукомплектованных деталей:

Дефектов раскроя:
Забракованных деталей:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
